Portfolio.html

```diff
@@ -93,10 +96,15 @@ <h2>Contact</h2>
 				<p>Send me an email or reach out on my social media using the buttons opposite.</p>
 			</div>	
 		</div>
-	</div> -->
+	</div> 
+
+And here's a link to the image for about me 
+<img src="AboutMeBG3.jpg" alt="Background to About Me" />
+
+-->
 
 <footer>
```

Comment:
Your footer is outside of your body tag. Also your missing the closing </html> tag at the very end of the document. Maybe try that tool for checking html syntax?

Reply:
Thanks Davin!